src/disk_interface.cc

```diff
@@ -71,7 +71,7 @@ TimeStamp TimeStampFromFileTime(const FILETIME& filetime) {
 
 TimeStamp StatSingleFile(const string& path, string* err) {
   WIN32_FILE_ATTRIBUTE_DATA attrs;
-  if (!GetFileAttributesExA(path.c_str(), GetFileExInfoStandard, &attrs)) {
+  if (!GetFileAttributesEx(Utf8ToWide(path).c_str(), GetFileExInfoStandard, &attrs)) {
```

Comment:
You might want to use the -W suffix everywhere since the code is written specifically for the wide char functions now.